### PR TITLE
LPS-175778 Inline Mail and Trash service.xml into ServiceBuilder

### PR DESCRIPTION
**Pull request by julschong**
Resubmitting for: https://github.com/brianchandotcom/liferay-portal/pull/130731
added XMLDTDVersionCheck SF suppression

### CODE REVIEW

File: modules/util/source-formatter-suppressions.xml [hunk 1]
```diff
@@ -18,6 +18,12 @@
 		<suppress checks="StringMethodsCheck" files="portal-tools-data-partitioning-sql-builder-postgresql/.*" />
 		<suppress checks="StringMethodsCheck" files="portal-tools-soy-builder/.*" />
 		<suppress checks="StringMethodsCheck" files="sass-compiler-dart/.*" />
+		<suppress checks="XMLDTDVersionCheck" files="portal-tools-service-builder/src/main/resources/com/liferay/mail/service\.xml" />
+		<suppress checks="XMLDTDVersionCheck" files="portal-tools-service-builder/src/main/resources/com/liferay/portlet/trash/service\.xml" />
```
Comment (julschong) [inline modules/util/source-formatter-suppressions.xml:22]: Added this here

File: modules/util/portal-tools-service-builder/src/main/resources/com/liferay/mail/service.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.0.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_0_0.dtd">
```
Comment (julschong) [inline modules/util/portal-tools-service-builder/src/main/resources/com/liferay/mail/service.xml:2]: Corrected the versions

File: modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portlet/trash/service.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0"?>
+<!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.0.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_0_0.dtd">
```
Comment (julschong) [inline modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portlet/trash/service.xml:2]: Corrected the version

**Comment by Kvale1733** [comment]
@julschong No regressions found. Testing passed on 7.0-7.3. SF issue is unrelated and is appearing in [acceptance](https://testray.liferay.com/home/-/testray/case_results/2206836362) also

**Comment by tinatian** [comment]
ci:forward

**Comment by tinatian** [comment]
ci:forward

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/131435

**Pull request closed without merging**